src/components/BotName.tsx

```diff
@@ -0,0 +1,10 @@
+import {useRouter} from "next/router";
+import {defaultBotName} from "../utils/constants";
+
+export default function BotName() {
+   const router = useRouter();
+   const botName = router.query.botName;
```

Comment:
shouldn't we use optional chaining `?` here?

Reply:
Not needed, query is always present, and in case of botName the fallback is the default name